lib/auth_web/controllers/people_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule AuthWeb.PeopleController do
+  @moduledoc """
+  Defines People controller functions
+  """
+  use AuthWeb, :controller
+
+  @doc """
+  `index/2` lists all the people who have authenticated with the auth app.
+  """
+  def index(conn, _params) do
+    # should be visible to superadmin and people with "admin" role
+    if conn.assigns.person.id == 1 do
+      render(conn, :index,
+        people: Auth.Person.list_people(),
+        roles: Auth.Role.list_roles(),
+        statuses: Auth.Status.list_statuses()
+      )
+
+      # Note: this can easily be refactored to save on DB queries. #HelpWanted
+    else
+      AuthWeb.AuthController.not_found(conn, "Only admins can see people ... for now!")
+    end
+  end
+
+  @doc """
+  `show/2` shows the profile of a person with all relevant info.
+  """
+  def show(conn, params) do
+    # should be visible to superadmin and people with "admin" role
+    if conn.assigns.person.id == 1 do
```

Comment:
> should be visible to superadmin and people with "admin" role

Is this still true or the comment outdated? Otherwise there still needs to be a mechanism for allowing people with an `admin` role

Reply:
Comment is still valid as we don't have the means for allowing _anyone_ to see the list of people who have authenticated. I think this could be a useful feature in the future, but for now it's restricted to the `superadmin` 👍